packages/cache-handler/src/cache-handler.ts

```diff
@@ -16,7 +16,19 @@ import type {
     TagsManifest,
 } from '@neshca/next-common';
 
+import { UseTtlOptions } from './common-types';
 import { isTagsManifest } from './helpers/is-tags-manifest';
+import { isStale } from './helpers/isStale';
+
+type PrerenderManifest = {
+    version: 4;
+    routes: Record<
+        string,
+        {
+            initialRevalidateSeconds: number | false;
+        }
+    >;
+};
```

Comment:
I assume this is where we store our parsed results?

Reply:
This file is created by Next.js during the building process